File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/TypeTemplateService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbTypeTemplate;

import entity.PageResult;

public interface TypeTemplateService {
	
	/**
	 * 查询所有品牌
	 */
	List<TbTypeTemplate> findAll();

	PageResult findPage(Integer pageNum, Integer pageSize);

	void add(TbTypeTemplate typeTemplate);

	TbTypeTemplate findOne(Long id);

	void update(TbTypeTemplate typeTemplate);

	void delete(Long[] ids);

	PageResult findPage(TbTypeTemplate typeTemplate, Integer pageNum, Integer pageSize);
	
	List<Map> findSpecList(Long id);
}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/typeTemplateService.js
// 品牌服务层
app.service('typeTemplateService',function($http){
	// 查询所有
	this.findAll = function(){
		return $http.get('../typeTemplate/findAll.do');
	}
	
	// 分页查询
	this.findPage = function(pageNum,pageSize){
		return $http.get("../typeTemplate/findPage.do?pageNum="+pageNum+"&pageSize="+pageSize);
	}
	
	// 添加
	this.add = function(entity){
		return $http.post('../typeTemplate/add.do',entity);
	}
	
	// 修改
	this.update = function(entity){
		return $http.post('../typeTemplate/update.do',entity);
	}
	
	// 根据id查
	this.findOne = function(id){
		return $http.get('../typeTemplate/findOne.do?id='+id);
	}
	
	// 删除
	this.dele = function(ids){
		return $http.get('../typeTemplate/delete.do?ids='+ids);
	}
	
	// 条件分页查询
	this.search = function(pageNum,pageSize,searchEntity){
		return $http.post("../typeTemplate/search.do?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity);
	}
	
	
})